controller/loginController.js

```diff
@@ -0,0 +1,113 @@
+const User = require('../models/user')
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt')
+const http = require('http-status-codes')
+const secretKey = require('../config/secretKey')
+const statusMsg = require('../config/statusMsg')
+const dotenv = require('dotenv')
+dotenv.config({
+    path: './config/.env'
+})
+const tokenGenerator = require('./authTokenGenerator')
+
+exports.login = async (req, res, next) => {
+    try {
+        if (req.body.email === '' || req.body.password === '') {
```

Comment:
@AwaleRohin you are using the express validator right??

Reply:
fixed